src/parser/shared/normalizers/FightEnd.js

```diff
@@ -0,0 +1,19 @@
+import EventsNormalizer from 'parser/core/EventsNormalizer';
+
+export const END_EVENT_TYPE = 'fightend';
+
+class FightEnd extends EventsNormalizer {
```

Comment:
Please add a comment above the class shortly explaining what it does and why, and optionally possible extensions.